### DISCUSSION TITLE
Option to concatenate input on ReplyKeyboardMarkup

### DISCUSSION THREAD

**Discussion by Alex23rodriguez** [aiogram/aiogram#1208] [Q&A]
**Is your feature request related to a problem? Please describe.**
I'd like to create a numpad using ReplyKeyboardMarkup. For this, I'd like each the message to accumulate each time I press a key and then send once I press enter (or a specific key) instead of sending each key individually

**Describe the solution you'd like**
an option can be passed when creating the ReplyKeyboardMarkup indicating that the text is not to be sent straight away.

**Describe alternatives you've considered**
The alternative is to use the normal keyboard, which isn't as convenient.

**Comment by JrooTJunior** [comment]
Please describe problem that you trying to solve, to help understand you.

Also i will transfer this issue to discussion.

**Comment by Alex23rodriguez** [comment]
As described, I'd like to implement a numpad using ReplyKeyboardMarkup. This is useful when answering texts that consist entirely of digits (and an optional period)
For example, the rows could look like this:

```python
ReplyKeyboardMarkup().row("7","8","9").row("4","5","6").row("1","2","3").row(".","0")
```
